### PR TITLE
What's New in SwiftWasm #2

### PR DESCRIPTION
**Pull request by MaxDesiatov**
The toolchain and upstream PRs sections are to be updated, but I guess the rest of the post is ready for review.

### CODE REVIEW

File: Content/posts/update-02-october-2020.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+practice it wasn't always clear which of these protocols was responsible for a specific conversion.
+After some deliberation, these were renamed to `ConstructibleFromJSValue` and `ConvertibleToJSValue`
+respectively in [#88](https://github.com/swiftwasm/JavaScriptKit/pull/88).
+
+[An issue was raised](https://github.com/swiftwasm/JavaScriptKit/issues/97) by
+[@yonihemi](https://github.com/yonihemi) this week on our JavaScriptKit repository about `i64`
+Wasm function return type support in Safari. The reason for it is that [Safari is the only major
+browser](https://webassembly.org/roadmap/) that doesn't support
+[Wasm `i64` to `BigInt` conversion](https://github.com/WebAssembly/JS-BigInt-integration).
+Unfortunately, there is a plenty of APIs that do require this conversion to work, and it's unclear
```
Comment (carson-katri) [inline Content/posts/update-02-october-2020.md]: Possibly change to
```suggestion
Unfortunately, there is a plethora of APIs that require this conversion to work, and it's unclear
```
or
```suggestion
Unfortunately, there are (plenty | lots | many) of APIs that require this conversion to work, and it's unclear
```

File: Content/posts/update-02-october-2020.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+yet if this can be polyfilled on the JavaScript side at all. Currently it looks like we need to
+apply some transformations to binaries produced by SwiftWasm to resolve this issue, but it remains
+to be seen how well that would work in practice.
+
+## Tokamak
+
+Tokamak didn't see major updates recently, but we've received some important bug reports during
+the last few weeks. Firstly, there's [an edge case with the `Picker`
```
Comment (carson-katri) [inline Content/posts/update-02-october-2020.md]:
```suggestion
the last few weeks. Firstly, there's [an edge case with `Picker`
views
```

**Review by carson-katri — COMMENTED** [review]
Some very minor grammar things, feel free to disregard them if they don't seem right.

File: Content/posts/update-02-october-2020.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+them fortnightly now. Let us know what you think of this new cadence. And here's a gentle reminder
+that this blog is fully open-source, so if you spot a typo, an error, a broken link, or have any
+other feedback, please feel free to file it [in our `blog.swiftwasm.org` GitHub
+repository](https://github.com/swiftwasm/blog.swiftwasm.org).
+
+## SwiftWasm Team Updates
+
+We would like to welcome [@yonihemi](https://github.com/yonihemi) to the SwiftWasm team who joined
+us in the beginning of October. After the previous contributions he made to
+[`carton`][https://carton.dev] it made perfect sense to expand our team. As always, we invite
```
Comment (j-f1) [inline Content/posts/update-02-october-2020.md]:
```suggestion
[`carton`](https://carton.dev) it made perfect sense to expand our team. As always, we invite
```

File: Content/posts/update-02-october-2020.md (hunk 1 above)
Comment (j-f1) [reply Content/posts/update-02-october-2020.md]: I agree with @carson-katri about “many:”

```suggestion
Unfortunately, there are many APIs that require this conversion to work, and it's unclear
```

**Review by kateinoigakukun — COMMENTED** [review]
We can note down that we updated swiftwasm-book for more practical usage. 👍

File: Content/posts/update-02-october-2020.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+dependency resolution due to strict checks that SwiftPM applies. If any package in your dependency
+tree contains unsafe flags, you can no longer depend on its semantic version, or a semantic version
+of any other package that depends on it.
+
+So far we were able to work around that with a hardcoded check for JavaScriptKit in our fork of
+SwiftPM. This was obviously a very ugly hack, but the biggest downside of that approach was that
+you couldn't depend on any package that had a semantic version dependency on JavaScriptKit in
+upstream Swift toolchains. That meant that libraries like [Tokamak](https://tokamak.dev) could not
+be built for macOS or Linux. And while the WebAssembly DOM renderer in Tokamak is the most useful
+module right now, this prevented us from working static HTML rendering in Tokamak on macOS and
```
Comment (yonihemi) [inline Content/posts/update-02-october-2020.md]: `prevented us from working` ? Not sure I follow the actual meaning.

Comment (MaxDesiatov) [reply Content/posts/update-02-october-2020.md]: Well spotted! Updated to "this prevented static HTML rendering from working on macOS and
Linux".

File: Content/posts/update-02-october-2020.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+yet if this can be polyfilled on the JavaScript side at all. Currently it looks like we need to
+apply some transformations to binaries produced by SwiftWasm to resolve this issue, but it remains
+to be seen how well that would work in practice.
+
+## Tokamak
+
+Tokamak didn't see major updates recently, but we've received some important bug reports during
+the last few weeks. Firstly, there's [an edge case with `Picker`
+views
+view](https://github.com/TokamakUI/Tokamak/issues/285) that use `\.self` as an identifier keypath.
```
Comment (j-f1) [inline Content/posts/update-02-october-2020.md]:
```suggestion
the last few weeks. Firstly, there's [an edge case with `Picker`
views](https://github.com/TokamakUI/Tokamak/issues/285) that use `\.self` as an identifier keypath.
```

**Review by yonihemi — APPROVED** [review]

**Review by j-f1 — APPROVED** [review]

**Review by carson-katri — APPROVED** [review]

**Review by yonihemi — APPROVED** [review]

**Pull request merged**